src/main/java/oshi/util/ExecutingCommand.java

```diff
@@ -35,6 +33,7 @@
 		} catch (IOException e) {
 			return null;
 		}
+		p.destroy();
```

Comment:
I don't think you can do this. The process spawns and then just hangs there and you don't stop it. You need to find a  way to run something that returns. Maybe using `top` isn't the best way? On Linux I believe /proc/stat can give you a lot of this, no idea about OSX.